src/package.rs

```diff
@@ -66,17 +59,25 @@ impl Package {
         }
     }
 
+    /// License of the package
     pub fn license(&self) -> &str {
         &self.license
     }
 
+    /// Succinct description of the package
     pub fn summary(&self) -> &str {
         &self.summary
     }
 
+    /// Longer description of the package
     pub fn description(&self) -> &str {
         &self.description
     }
+
+    /// Buildtime of the package
+    pub fn buildtime(&self) -> i32 {
```

Comment:
Is there a better type that could be used for this other than `i32`? Possibly `SystemTime`?

Regardless it might make sense to at least add a type alias.

Reply:
Seems that is possible to use `time::SystemTime`, and use it properly later on. That was a good idea.

Reply:
Done.